### PR TITLE
[CI] Uplift Linux GPU RT version to 23.13.26032.30

### PR DESCRIPTION
**Pull request by dm-vodopyanov**
This patch updates GPU RT version on Linux to support 
[DEVICE_IP_VERSION_EXT](https://github.com/oneapi-src/level-zero-spec/pull/23) implementation required for https://github.com/intel/llvm/pull/9843.

### CODE REVIEW

File: devops/dependencies.json [hunk 1]
```diff
@@ -74,4 +74,4 @@
       "root": "{ARCHIVE_ROOT}/comp/oclfpga/win"
     }
   }
-}
\ No newline at end of file
+}
```
Comment (dm-vodopyanov) [inline devops/dependencies.json:77]: The line was automatically added by Github editor, if it shouldn't be modified, please let me know.

**Comment by dm-vodopyanov** [comment]
@intel/dpcpp-devops-reviewers, friendly ping

**Comment by aelovikov-intel** [comment]
Please merge in latest `origin/sycl`. I believe I've fixed pre-commit for drivers update in https://github.com/intel/llvm/pull/10071.

**Comment by dm-vodopyanov** [comment]
> Please merge in latest `origin/sycl`. I believe I've fixed pre-commit for drivers update in #10071.

Thanks, done

**Comment by dm-vodopyanov** [comment]
@aelovikov-intel @intel/dpcpp-devops-reviewers, as there was no any pre-commit issues, and this driver update is required to unblock https://github.com/intel/llvm/pull/9843, can it be approved and merged?

**Review by bader — APPROVED** [review]
I think PRs like this can be merged automatically if they pass all tests. There is nothing to review here.

NOTE: There is newer driver version already, but two tests fail in CI. Please, take a look at https://github.com/intel/llvm/pull/10087.

**Comment by dm-vodopyanov** [comment]
Post-commit failure:
```
Unexpectedly Passed Tests (1):
  SYCL :: ESIMD/accessor_local.cpp
```

**Pull request merged**